### PR TITLE
add instruction about how to get Suzanne model

### PR DESCRIPTION
**Pull request by Clarkszw**
update the instruction to online Getting Started page.
First PR#761 is not reflected in the file. This is the second attempt.

### CODE REVIEW

File: docs/source/getting_started.rst [hunk 1]
```diff
@@ -10,7 +10,18 @@ Start by importing FURY.
...
+
+We can download the model by
+
+.. code-block:: python
+
+    from fury.data import fetch_viz_models
+    fetch_viz_models()
+    
+Then the models will be downloaded in ``~/.fury/models``.
+To include the ``suzanne`` model used by Blender:
```
Comment (skoudoro) [inline docs/source/getting_started.rst:25]: Can you add how to get the `suzanne` file path from the fetcher ? 

Thank you

**Review by skoudoro — COMMENTED** [review]
Looks good to me! See below an additional comment.

Thanks in advance for addressing it

File: docs/source/getting_started.rst [hunk 2]
```diff
@@ -10,7 +10,34 @@ Start by importing FURY.
...
+We can download the model by
+
+.. code-block:: python
+
+    from fury.data import fetch_viz_models
+    fetch_viz_models()
+    
+Then the models will be downloaded in ``~/.fury/models``
+
+If store the return value to ``models``, you can find the detailed information in the dictionary. The directory of the ``suzanne`` model will be in ``model[1]``.
```
Comment (skoudoro) [inline docs/source/getting_started.rst:25]: typo: if ?

typo:  models[1].    `s` missing to be coherent with the code below

**Review by skoudoro — COMMENTED** [review]
Overall, Looks good, Thanks! 

see below some comments.

After that, I hope the merging will be ok this time

File: docs/source/getting_started.rst (hunk 2 above)
Comment (Clarkszw) [reply docs/source/getting_started.rst]: love the process of polishing:) done👍

**Comment by skoudoro** [comment]
thank you @Clarkszw, merging

**Pull request merged**